yogstation/code/game/world.dm

```diff
@@ -41,7 +41,7 @@ GLOBAL_LIST_EMPTY(donators)
 	var/server_name = CONFIG_GET(string/servername)
 	if (server_name)
 		s += "[server_name]\] &#8212; Dive in Now: Perfect for Beginners!"
-	s += "<br>99% Lag-Free <b>MRP</b> Experience! Join the Adventure!"
+	s += "<br>99% Lag-Free <b>Regular Events!!</b> Join the Adventure!"
```

Comment:
```suggestion
	s += "<br>99% Lag-Free <b>Regular Events!</b> Experience! Join the Adventure!"
```

The council vote states Remove the MRP tag on BYOND, and instead replace it with "Regular Events!". This change removes the phrase "MRP" and replaces it with "Regular Events!". Removal of other words requires a separate council vote.